ONSdigital/ssdc-rm-support-tool
# 158 spike support tool improve background api call behaviour

# Motivation and Context
interval refresh every 1 second in dozens of places would with multiple users means 1000s of API calls per minute and would make audit logging a lot more noisy.

# What has changed
Have removed 90% of the interval refreshes and replaced with simple updates if you for example add or remove a user.  Only left in for bulk/sample uploads and exception manager - where interval probably makes sense.

# How to test?
make it, build it.  Start up docker dev with new Image.  Play with it,  does it still work as you'd expect?

# Links
https://trello.com/c/UxU0imY7/158-spike-support-tool-improve-background-api-call-behaviour-1-day
